presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/SemanticException.java

```diff
@@ -39,11 +39,15 @@ public SemanticException(SemanticErrorCode code, Node node, String format, Objec
 
     public SemanticException(SemanticErrorCode code, Optional<NodeLocation> location, String format, Object... args)
     {
-        super(formatMessage(format, location, args));
-        requireNonNull(code, "code is null");
+        this(code, null, location, format, args);
```

Comment:
@jainxrohit I couldn't comment on line 37 above, but can you explain when node would be null? I looked through https://github.com/prestodb/presto/pull/18762, and I couldn't figure out.